### ISSUE TITLE
cockpit-composer PF4 migration - package lists

### ISSUE THREAD

**Issue by larskarlitski** [weldr/cockpit-composer#804]
UI in Composer should be migrated to use PatternFly 4 UI components, to improve consistency with other products and take advantages of accessibility improvements.

As a primary focus for migration, all PF3 list views should be migrated to PF4 data list components. For this proposal, the focus will be on lists showing packages. This includes the following:

- Blueprint and Edit blueprint pages - list view of components / dependencies
- Blueprint and Edit blueprint pages - component details view in a panel - list of dependencies
- Blueprint and Edit blueprint pages - component details in an expanded list item - list of dependencies
- Edit blueprint page - list view of available components

The list of package updates in the Pending Changes modal will be included in a later proposal.
